include/ccf/service/node_info_network.h

```diff
@@ -75,15 +79,21 @@ namespace ccf
     NetInterface node_to_node_interface;
     RpcInterfaces rpc_interfaces;
   };
+
   DECLARE_JSON_TYPE_WITH_OPTIONAL_FIELDS(NodeInfoNetwork_v2::NetInterface);
   DECLARE_JSON_REQUIRED_FIELDS(NodeInfoNetwork_v2::NetInterface, bind_address);
   DECLARE_JSON_OPTIONAL_FIELDS(
     NodeInfoNetwork_v2::NetInterface,
     endorsement,
     max_open_sessions_soft,
     max_open_sessions_hard,
-    published_address);
-  DECLARE_JSON_TYPE(NodeInfoNetwork_v2);
+    published_address,
+    protocol,
+    http_configuration);
+
+  DECLARE_JSON_TYPE(NodeInfoNetwork_v2::ACME);
```

Comment:
This is what breaks the build in the absence of the ACME client PRs. We do not intend to backport the ACME client to 2.0.x, it's a 3.x feature, so this needs editing out.

Reply:
Ah yes, the merge was awkward. This should now work.